pkg/manager/member/tikv_member_manager.go

```diff
@@ -725,7 +729,9 @@ func (tkmm *tikvMemberManager) setStoreLabelsForTiKV(tc *v1alpha1.TidbCluster) (
 		if !tkmm.storeLabelsEqualNodeLabels(store.Store.Labels, ls) {
 			set, err := pdCli.SetStoreLabels(store.Store.Id, ls)
 			if err != nil {
-				klog.Warningf("failed to set pod: [%s/%s]'s store labels: %v", ns, podName, ls)
+				evtStr := fmt.Sprintf("failed to set pod: [%s/%s]'s store labels: %v", ns, podName, ls)
+				tkmm.recorder.Event(tc, corev1.EventTypeWarning, FailedSetStoreLabels, evtStr)
+				klog.Warning(evtStr)
```

Comment:
```suggestion
```
this is not necessary, the recorder will print logs too.

Reply:
fixed